blade.md

```diff
@@ -72,67 +72,67 @@ When defining a child view, use the Blade `@extends` directive to specify which
     @endsection
 
-In this example, the `sidebar` section is utilizing the `@@parent` directive to append (rather than overwriting) content to the layout's sidebar. The `@@parent` directive will be replaced by the content of the layout when the view is rendered.
+在上面的例子中，`sidebar` 区块利用 `@parent` 指令追加布局中的 sidebar 区块中的内容，如果不使用则会覆盖掉布局中的这部分内容。 `@parent` 指令会在视图被渲染时替换为布局中的内容。
 
-Blade views may be returned from routes using the global `view` helper:
+Blade 视图可以通过在路由中使用全局辅助函数 `view` 方法来返回：
 
     Route::get('blade', function () {
         return view('child');
     });
 
 <a name="displaying-data"></a>
-## Displaying Data
+## 展示数据
 
-You may display data passed to your Blade views by wrapping the variable in curly braces. For example, given the following route:
+你可以使用中括号给变量视图中的变量复制。例如， 如下面的路由设置：
 
     Route::get('greeting', function () {
         return view('welcome', ['name' => 'Samantha']);
     });
 
-You may display the contents of the `name` variable like so:
+你可以在视图中这样来输出 `name` 变量的内容：
 
     Hello, {{ $name }}.
 
-Of course, you are not limited to displaying the contents of the variables passed to the view. You may also echo the results of any PHP function. In fact, you can put any PHP code you wish inside of a Blade echo statement:
+当然， 你并没有被限制只允许显示传递到视图中的变量的内容。你也可以从原生 PHP 方法中返回内容。事实上，你可以在 Blade 中 echo 任意的 PHP 代码：
 
     The current UNIX timestamp is {{ time() }}.
 
-> {note} Blade `{{ }}` statements are automatically sent through PHP's `htmlentities` function to prevent XSS attacks.
+> {note} Blade `{{ }}` 声明中的内容是自动通过 PHP 的 `htmlentities` 方法过滤的，用以避免 XSS 的攻击。
 
-#### Echoing Data If It Exists
+#### 输出数据假如它存在
 
-Sometimes you may wish to echo a variable, but you aren't sure if the variable has been set. We can express this in verbose PHP code like so:
+有时候你可能希望输出一个变量，但是你并不确定这个变量是否已经被定义，你可以使用 PHP 的这个表达式：
 
     {{ isset($name) ? $name : 'Default' }}
 
-However, instead of writing a ternary statement, Blade provides you with the following convenient short-cut, which will be compiled to the ternary statement above:
+事实上，Blade 提供了更便捷的方式来代替这种三元运算：
 
     {{ $name or 'Default' }}
 
-In this example, if the `$name` variable exists, its value will be displayed. However, if it does not exist, the word `Default` will be displayed.
+在这个例子中，如果变量 `$name` 存在， 它的值将被输出显示。 但是，如果它不存在，将会显示 `Default` 。
 
-#### Displaying Unescaped Data
+#### 显示未转义的数据
 
-By default, Blade `{{ }}` statements are automatically sent through PHP's `htmlentities` function to prevent XSS attacks. If you do not want your data to be escaped, you may use the following syntax:
+默认的，Blade `{{ }}` 声明会自动的使用 PHP 的 `htmlentities` f方法来转义数据避免 XSS 的攻击。如果你不想你的数据被转义，你可以使用下面的语法：
 
     Hello, {!! $name !!}.
 
-> {note} Be very careful when echoing content that is supplied by users of your application. Always use the escaped, double curly brace syntax to prevent XSS attacks when displaying user supplied data.
+> {note} B当你在应用中输出用户输入的数据时应该非常的谨慎，你应该总是使用 `{{  }}` 语法来转义内容中的任何的 HTML 实体。
```

Comment:
Be very careful when echoing content that is supplied by users of your application. Always use the escaped, double curly brace syntax to prevent XSS attacks when displaying user supplied data.

这一句最重要的关键词是 XSS 攻击，需要翻译处理，我的建议翻译是：

    你应该总是使用 `{{  }}` 语法来输出用户提交的数据，这样能最有效的让你的程序免于 XSS 攻击。